remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,13 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, *args):
+        """
+        Storing the Event details using the logger.
+        """
+        event_line = ",".join("{}".format(arg) for arg in args)
```

Comment:
When I add the new handler "hdinsighteventshandler" the method there will be the same and it will help to insert them in the MDS table with the same arrangement. 

Let's say we have dim1, dim2..etc. arranged in this way then, we put them in a key-value, we will need to iterate over them to insert them in the right way. 

So, if they are already sorted in a list? so we will insert them easily without need to iterate over the dictionary to put the value for each attribute in the right place.